README.md

```diff
@@ -9,6 +9,27 @@
 
 # W24 Production Deployments
 
+## Partial pitest runs
+
+This repo has support for partial pitest runs
+
+For example, to run pitest on just one class, use:
+
+```
+mvn pitest:mutationCoverage -DtargetClasses=edu.ucsb.cs156.example.controllers.RestaurantsController
+```
+
+To run pitest on just one package, use:
+
+```
+mvn pitest:mutationCoverage -DtargetClasses=edu.ucsb.cs156.example.controllers.*
```

Comment:
```suggestion
mvn pitest:mutationCoverage -DtargetClasses=edu.ucsb.cs156.example.controllers.\*
```

The backslash is important.

Also, please change the examples to ones that actually work on the organic codebase; these are just copy/pasted from STARTER-team03.

Reply:
Appologies, you said copy over so I thought you meant literally copy over.